paymentsheet/src/test/java/com/stripe/android/customersheet/utils/CustomerSheetTestHelper.kt

```diff
@@ -112,6 +112,7 @@ internal object CustomerSheetTestHelper {
             isLiveModeProvider = { isLiveMode },
             logger = Logger.noop(),
             confirmationHandlerFactory = createTestConfirmationHandlerFactory(
+                instanceId = "CustomerSheet",
```

Comment:
Let's update all of these test values to contain test in their id. There's others as well.